README.md

```diff
@@ -1 +1,649 @@
+│  │  fct: nextReqECDH         │  │
+│  │  att: []                  │  │
+│  │  prf: ...                 │  │
+│  │                           │  │
+│  └───────────────────────────┘  │
+│                                 │
+└─────────────────────────────────┘
+```
+
+# 3.5 Responder Acknowledgment
+
+The Responder MUST respond with an acknowledgment that the challenge in [§3.4](#34-requestor-challenge) was accepted.
+
+```
+Requestor                  Responder
+    ⋮                          ⋮
+    │           ACK            │ (5)
+    │◄─────────────────────────┤
+    │                          │
+```
+
+### 3.5.1 Payload
+
+| Field  | Value                             | Description                                                    | Required |
+| ------ | --------------------------------- | -------------------------------------------------------------- | -------- |
+| `awv`  | `"0.1.0"`                         | AWAKE message version                                          | Yes      |
+| `type` | `"awake/msg"`                     | Generic AWAKE message type                                     | Yes      |
+| `id`   | `sha3_256(resEcdhPk + reqEcdhPk)` | Message ID                                                     | Yes      |
+| `iv`   |                                   | Initialization vector for the encrypted payload                | Yes      |
+| `msg`  |                                   | Fulfilled challenge payload encrypted with Step 4 ECDH AES-key | Yes      |
+
+#### 3.5.1.1 Encrypted Message
+
+The encrypted message payload MUST include an `awake/ack` field, with a value of the requestor's long-term DID. This payload MAY contain additional fields. This is often useful if dovetailing the ACK with the first message of a session using the 
+
+``` javascript
+{
+  ...,
+  "awake/ack": reqActualDid
+}
+```
+
+## 4 Secure Session
+
+```
+Requestor                  Responder
+    ⋮                          ⋮
+    │                          │
+    │         Messages         │ (6)
+    │◄────────────────────────►│
+    │                          │
+    ⋮                          ⋮
+```
+
+Messages sent over an established AWAKE session MUST contain the following keys:
+ 
+| Field  | Value                                                 | Description                                                             | Required |
+| ------ | ----------------------------------------------------- | ----------------------------------------------------------------------- | -------- |
+| `awv`  | `"0.1.0"`                                             | AWAKE message version                                                   | Yes      |
+| `type` | `"awake/msg"`                                         | Generic AWAKE message type                                              | Yes      |
+| `mid`  | `sha3_256(latestSenderEcdhPk + latestReceiverEcdhPk)` | Message ID                                                              | Yes      |
```

Comment:
Is it worth linking to the relevant sections for "Message ID" and "Session ID" in this and the other tables?